src/dimred/lsi/config.vsh.yaml

```diff
@@ -0,0 +1,100 @@
+functionality:
+  name: lsi
+  namespace: "dimred"
+  description: |
+    Runs Latent Semantic Indexing. Computes cell embeddings, feature loadings and singular values. Uses the implementation of scipy.
+  arguments:
+    # inputs
+    - name: "--input"
+      alternatives: ["-i"]
+      type: file
+      description: Input h5mu file
+      direction: input
+      required: true
+      example: input.h5mu
+
+    - name: "--modality"
+      type: string
+      default: "atac"
+      required: false
+
+    - name: "--layer"
+      type: string
+      description: Use specified layer for expression values instead of the .X object from the modality.
+      required: false
+
+    - name: "--var_input"
+      type: string
+      description: Column name in .var matrix that will be used to select which genes to run the LSI on.
+      required: false
+
+    # outputs
+    - name: "--output"
+      alternatives: ["-o"]
+      type: file
+      description: Output h5mu file.
+      direction: output
+      required: true
+      example: output.h5mu
+
+    - name: "--output_compression"
```

Comment:
The compression? f.ex. [UMAP](https://github.com/openpipelines-bio/openpipeline/blob/main/src/dimred/umap/config.vsh.yaml#L39) and [PCA](https://github.com/openpipelines-bio/openpipeline/blob/main/src/dimred/pca/config.vsh.yaml#L43) do it

Reply:
This argument is part of almost all components that output a h5mu component. We use compression at every step of the workflows where we publish data because uncompressed h5mu's can become quite large. Keep in mind that enabling compression does not compress the whole h5mu file (similar to using `gzip output.h5mu > output.h5mu.gz`) but uses compression that is provided by the h5 format where parts of the data in the file is compressed. Data is decompressed seemlessly by the h5 library when the file is read. See https://docs.h5py.org/en/stable/high/dataset.html#filter-pipeline

Reply:
Thanks! Then I should add it to the components I created 😅